src/ResourceManager/Resources/Commands.Resources/help/Get-AzureRmManagementGroup.md

```diff
@@ -0,0 +1,233 @@
+---
+external help file: Microsoft.Azure.Commands.ManagementGroups.dll-Help.xml
+Module Name: AzureRM.ManagementGroups
+online version: 
```

Comment:
You will need to fill out all these fields for all help files - should be https://docs.microsoft.com/en-us/powershell/module/azurerm.managementgroups/<cmdletname>